test/shared/src/main/scala/zio/test/DefaultTestReporter.scala

```diff
@@ -38,10 +38,12 @@ object DefaultTestReporter {
                          case Spec.TestCase(_, test) => test.map(_.isLeft);
                          case _                      => UIO.succeed(false)
                        })
-            hasFailures   = failures.exists(identity)
-            status        = if (hasFailures) Failed else Passed
-            renderedLabel = if (hasFailures) renderFailureLabel(label, depth) else renderSuccessLabel(label, depth)
-            rest          <- UIO.foreach(specs)(loop(_, depth + tabSize)).map(_.flatten)
+            hasFailures = failures.exists(identity)
+            status      = if (specs.isEmpty) Ignored else if (hasFailures) Failed else Passed
```

Comment:
What is the significance of returning `Ignored` versus passed here? It looks like we get the same output where we don't run any tests and exit with a success code.

Reply:
You're right, it doesn't seem to matter.